EAD/Session 1/Assignment1/index.html

```diff
@@ -0,0 +1,505 @@
+<!DOCTYPE html>
+<html lang = "en">
+<head>
+    <meta charset = "UTF-8">
+    <title>Document</title>
+    <style>

```

Comment:
We can use External Style rather than internal style for CSS

Reply:
ok